data/import/foodscapes/README.md

```diff
@@ -0,0 +1,16 @@
+# Creating a SQLite db from Foodscapes CSV data
+
+This should be set up to run in a Docker container: as a first quick step,
+however, the following steps should be all that is needed. The only dependency
+is the `sqlite3` binary.
+
+```
+sudo apt install sqlite3 # or whatever appropriate to install sqlite3 on your OS
```

Comment:
I will add this to make it super straightforward dummy proven to setup :sweat_smile: 
```suggestion
sudo apt install sqlite3 # or whatever appropriate to install sqlite3 on your OS
mkdir source
mkdir dest
```

Reply:
good point. I have opted for pre-creating the folders (fd8889e67b92d6cbe18950bde2eee6cab0f24717) while keeping them gitignored: I like having the directory structure ready and visible, in fact 😄